src/rules.ts

```diff
@@ -268,8 +268,10 @@ class MatchingRules extends Array<MatchingRule> {
 
     const matchedRules = await Promise.all(matchingRules);
 
-    const filtered = matchedRules.filter((rule) => isMatchingRule(rule) && rule.matched);
+    const alwaysMatchedRulesTypes = [RuleActions.status]
```

Comment:
this const name is causing me issues. 

Can't come up with a better name for it. maybe @cyamonide can help?

Reply:
instead of _always matched_, how about _mandatory_ or _required_? first name that came to mind was `mandatoryRulesTypes`.

Reply:
Yea. @andrecrts mind changing it?

Reply:
changed